tests/test

```diff
@@ -24,7 +24,7 @@ if [ $(basename $(sudo readlink /proc/1/exe)) == systemd ] ; then
 fi
 
 STAGE1_SRC_FROM_HOST=0
-if tar tvf ../bin/stage1.aci rootfs/flavor|grep -q usr-from-host ; then
+if tar xOvf ../bin/stage1.aci manifest|grep -q usr-from-host ; then
```

Comment:
in GNU tar it is "extract to standard output", since there is no "broken symlink" anymore i grep the manifest instead for the flavour value, to bail out CI testing for the usr-from-host flavour as before.